modules/gov/handle_msg.go

```diff
@@ -108,39 +104,44 @@ func (m *Module) handleMsgSubmitProposal(tx *juno.Tx, index int, msg *govtypesv1
 		return fmt.Errorf("error while storing proposal recipient: %s", err)
 	}
 
+	// Unpack the proposal interfaces
+	err = proposal.UnpackInterfaces(m.cdc)
+	if err != nil {
+		return fmt.Errorf("error while unpacking proposal interfaces: %s", err)
+	}
```

Comment:
Using codec to unpack Any type Content to the underlying structure, then we can get proposal content by proposal.GetContent() without error.